css/wee.mixins.less

```diff
@@ -2425,9 +2425,9 @@
 \*------------------------------------*/
 
 .transform (...) {
-	-ms-transform: @arguments;
-	-webkit-transform: @arguments;
-	transform: @arguments;
+	-ms-transform+_: @arguments;
```

Comment:
It is just a Less feature I hadn't previously been aware of. You can do something like the following now and the output concatenate instead of the second overriding. Very handy.

```
.rotate(45);
.scale(1.1);
```